.mailmap

```diff
@@ -213,6 +214,7 @@ Abhinav Cillanki <abhinavcillanki@kgpian.iitkgp.ac.in> JebronLames32 <abhinavcil
 Abhishek <uchiha@pop-os.localdomain>
 Abhishek Chaudhary <ac5003@columbia.edu>
 Abhishek Garg <abhishekgarg119@gmail.com>
+ABHISHEK KUMAR <96915839+abhishek-kuma@users.noreply.github.com> ABHISHEK KUMAR <abhishekkumaruprama1@gmail.com> 
```

Comment:
It should be like this:
`ABHISHEK KUMAR <abhishekkumaruprama1@gmail.com> ABHISHEK KUMAR <96915839+abhishek-kuma@users.noreply.github.com>`